"no field `ld_args` on type `pkg_config::Library`" on nightly

running a [CI pipeline](https://github.com/dimbleby/rust-c-ares/actions/runs/9565581214/job/26368816320?pr=108) against nightly rust

```
error[E0609]: no field `ld_args` on type `pkg_config::Library`
    --> /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/system-deps-7.0.0/src/lib.rs:1068:24
     |
1068 |             ld_args: l.ld_args,
     |                        ^^^^^^^ unknown field
     |
     = note: available fields are: `libs`, `link_paths`, `frameworks`, `framework_paths`, `include_paths` ... and 2 others

For more information about this error, try `rustc --explain E0609`.
error: could not compile `system-deps` (lib) due to 1 previous error
```
maybe you already know about this, maybe this is a nightly rust problem and not a system-deps problem, certainly it's not important to me so long as it is happening only on nightly - but I figured I should let you know about it in case work is needed

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -20,7 +20,7 @@ documentation = "https://docs.rs/system-deps/"
 readme = "README.md"
 
 [dependencies]
-pkg-config = "0.3.23"
+pkg-config = "0.3.25"
 toml = { version = "0.8", default-features = false, features = ["parse"] }
 version-compare = "0.2"
 heck = "0.5"

